### PR TITLE
1.4.development environment configuration

### PR DESCRIPTION
**Pull request by GinezisNWD**
## PR Type

What kind of change does this PR introduce?

```
- [ ] Feature
- [ ] Bugfix
- [ ] Refactoring
- [ ] Documentation changes
- [ ] Tests
- [x] Other
```

## What's new?
setup tsconfig, webpack-config, eslint-config, prettier, jest-config.

Please include a summary of the changes.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dist
+node_modules
+webpack**
```
Comment (miroslav-zarenkov) [inline .eslintignore]: Add last empty line please, here and in all other files.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,11570 @@
...
+        "url": "https://github.com/sponsors/sindresorhus"
+      }
+    },
+    "node_modules/jest-circus": {
+      "version": "29.6.2",
+      "resolved": "https://registry.npmjs.org/jest-circus/-/jest-circus-29.6.2.tgz",
+      "integrity": "sha512-G9mN+KOYIUe2sB9kpJkO9Bk18J4dTDArNFPwoZ7WKHKel55eKIS/u2bLthxgojwlf9NLCVQfgzM/WsOVvoC6Fw==",
+      "dev": true,
+      "dependencies": {
+        "@jest/environment": "^29
```
Comment (miroslav-zarenkov) [inline package-lock.json:11570]: Add this file to .gitignore.

**Review by miroslav-zarenkov — CHANGES_REQUESTED** [review]
Please fix all.

File: package-lock.json (hunk 2 above)
Comment (GinezisNWD) [reply package-lock.json:11570]: fixed

**Review by toronto-tokyo — APPROVED** [review]
All is ok

**Review by miroslav-zarenkov — APPROVED** [review]
Looks good to me!

**Pull request merged**